src/runtime/runtime-test.js

```diff
@@ -1515,4 +1486,58 @@ describes.realWin('ConfiguredRuntime', {}, env => {
   it('should return events manager', () => {
     expect(runtime.eventManager() instanceof ClientEventManager).to.be.true;
   });
+
+  describe('configPromise', () => {
```

Comment:
one more test that could be added here, if you think useful, is to demonstrate how one may attach a filterer before resolving a promise and as a result have the filterer do its magic on events that are queued that should be filtered.

For example.

stub filterer registration api in event manager to save filterer 
stub logEvent method to count events after the saved filterer is invoked
create configured runtime with a promise
get event manager from it and attach filterer
queue 2 events
check event count is 1

Often test code serves as documentation on use cases supported by an API surface. An example of this sort is a good reminder as to why did it this way documented by way of test code. IMHO

Reply:
Good idea, done